databricks/koalas/series.py

```diff
@@ -4962,6 +4965,76 @@ def tail(self, n=5):
         """
         return first_series(self.to_frame().tail(n=n))
 
+    def product(self, min_count=0):
+        """
+        Return the product of the values.
+
+        .. note:: unlike pandas', Koalas' emulates product by ``exp(sum(log(...)))``
+            trick. Therefore, it only works for positive numbers.
+
+        .. note:: the result can be slightly difference from pandas since the rounding error.
```

Comment:
I think it's okay to don't mention this. Fractions can be not exactly matched anyway at Python in general.